target_api/client.py

```diff
@@ -54,17 +56,21 @@ def endpoint(self) -> str:
     @property
     def unified_schema(self) -> BaseModel:
         return None
-    
+
+    def response_error_message(self, response: requests.Response) -> str:
+        try:
+            response_text = f" with response body: {response.text}"
+        except:
+            response_text = None
+        return f"Status code: {response.status_code} with {response.reason} for path: {response.request.url} with response body: {response_text}"
```

Comment:
Won't this:
```
with response body: {response_text}
```
log
```
with response body: with response body: {response.text}
``` 
based on line 62?

Reply:
Resolved.